addons/sourcemod/scripting/superzombiefortress.sp

```diff
@@ -5615,6 +5597,69 @@ public MRESReturn Client_GetMaxHealth(int iClient, Handle hReturn)
 	return MRES_Ignored;
 }
 
+public MRESReturn Client_OnGiveNamedItem(int iClient, Handle hReturn, Handle hParams)
+{
+	static int iBlockedIndexes[] = {
+		57,  //Razorback
+		231, //Darwin's Danger Shield
+		642, //Cozy Camper
+		133, //Gunboats
+		444, //Mantreads
+		405, //Ali Baba's Wee Booties
+		608  //The Bootlegger
+	};
+	
+	char sClassname[256];
+	DHookGetParamString(hParams, 1, sClassname, sizeof(sClassname));
+	
+	int iDefIndex = DHookGetParamObjectPtrVar(hParams, 3, 4, ObjectValueType_Int) & 0xFFFF;
+	int iSlot = TF2Econ_GetItemSlot(iDefIndex, TF2_GetPlayerClass(iClient));
```

Comment:
`TF2Econ_GetItemSlot` itself give incorrect slots for engi and spy PDA. Need to add new stock for this, get econ slot then convert to actual in-game slot

Reply:
It doesn't matter if we get the incorrect slots for PDAs though? We have no reason to block engie's PDA and spy's disguise kit gets blocked anyway.

Reply:
Should still work like that, but i feel uncomfortable with getting incorrect slots while still works like that.